### PR TITLE
[DPMMA-2044] Builder custom fonts

### PR DESCRIPTION
**Pull request by annamunk**
Docs for https://github.com/mautic/grapesjs-preset-mautic/pull/43 and https://github.com/mautic/mautic/pull/12067

### CODE REVIEW

File: docs/builders/email_landing_page.rst [hunk 1]
```diff
@@ -127,6 +127,35 @@ If you search through the list of available Themes, the new MJML Themes ``Brienz
 
 To learn more about creating Themes please :doc:`check the documentation</builders/creating_themes>`. 
 
+Custom fonts
+************
+
+The Style Manager > Typography > Fonts list can be extended to include custom fonts.
```
Comment (RCheesley) [inline docs/builders/email_landing_page.rst:133]:
```suggestion
From Mautic 4.4.9 you can extend the Style Manager > Typography > Fonts list to include custom fonts.
```

Vale fix for grammar, and also mentioning the version from which you will be able to do this (assuming it goes into 4.4.9 and also 5.x).

File: docs/builders/email_landing_page.rst [hunk 2]
```diff
@@ -127,6 +127,35 @@ If you search through the list of available Themes, the new MJML Themes ``Brienz
...
+
+.. image:: images/editorfonts.jpg
+  :width: 280
+  :alt: Screenshot of the Fonts in Style Manager > Typography
+
+New options are defined as elements of the ``'editor_fonts'`` array in the local configuration file (most cases ``app/config/local.php``). The font should have a unique name and a valid css style url. See example below:
```
Comment (RCheesley) [inline docs/builders/email_landing_page.rst:139]:
```suggestion
You define options as elements of the ``'editor_fonts'`` array in the local configuration file - in most cases located in ``app/config/local.php``. The font should have a unique name and a valid CSS style URL. See example below:
```

Vale fix for active voice.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for making the PR @annamunk! Just a couple of fixes suggested by the linter to change the text from passive to active, and I think we should mention the version from which these changes will be applied.

**Comment by annamunk** [comment]
@RCheesley thank you! I will add suggested the fixes. 
In the Mautic repository, the changes seem to have been merged into version 5.x. Should I prepare there related PR with this functionality for 4.x branch, so it can be released from version 4.4.9?

**Comment by annamunk** [comment]
@RCheesley I corrected the documentation. Could you please take a look again?

**Review by RCheesley — APPROVED** [review]

**Comment by RCheesley** [comment]
Thanks @annamunk - merged on the preset repo and the mautic/mautic repo is already merged, so good to go!

**Pull request merged**